national-institute-of-technology-karnataka.csl

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```

Comment:
If you expect the style to only be used in British English, it's better to restore the `default-locale="en-GB"` on the `<style>` tag. That way users won't have to select the language by hand each time.

(you can update this pull request through the instructions given above: https://github.com/citation-style-language/styles/pull/2370#issuecomment-267961802)